coralproject/talk
# [CORL-859] Convert Featured stream into Answered for Q&A

## What does this PR do?

Converted the feature tab and stream into a question/answer combo `Answered` stream tab.

## How do I test this PR?

- Convert a story to Q&A
- Set an expert
- Create a question with a normal commenter
- Answer it with the expert commenter account
- See the question/answer combo appear in the `Answered` stream/tab
